release-content/0.14/migration-guides/12250_Batching_replace_GpuArrayBufferIndexindex_with_a_u32.md

```diff
@@ -0,0 +1 @@
+`GpuArrayBuferIndex::index` is now a u32 instead of a `NonMaxU32`. Remove any calls to `NonMaxU32::get` on the member.
```

Comment:
```suggestion
`GpuArrayBuferIndex::index` is now a u32 instead of a `NonMaxU32`. Remove any calls to `NonMaxU32::get` on the member.
```
```suggestion
`GpuArrayBufferIndex::index` is now a u32 instead of a `NonMaxU32`. Remove any calls to `NonMaxU32::get` on the member.
```